tasks/test.js

```diff
@@ -84,6 +84,14 @@ module.exports = function (grunt) {
     'stop:apiTestServer'
   ]);
 
+  grunt.registerTask('test:app', [
```

Comment:
`test:app:integration` or `test:integration:app` maybe?

Reply:
I'm just following the pattern that already exists for api and "functional" tests with this naming convention.  We can revisit the naming convention across the board, but let's not do that in this PR.